### PR TITLE
[TestChallenge]Update/add - pageobjects and specs

### PR DESCRIPTION
**Pull request by sumitbasra**
## Description
Extend Test Challenge

Updated:
`home.page.js` - added elements and methods.
`home.spec.js` - added more tests.

Added:
`custom.page.js` - to handle all custom methods.
`empire.building.page.js` - pageobject for Empire State Building page
`empire.building.spec.js` - test to validate Empire State Building page

## Screenshot
Screen Shot 2020-05-14 at 7 26 56 PM

### CODE REVIEW

File: test/pageobjects/empire.building.page.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Page from './page'
+
+class EmpireStatePage extends Page {
+    // Elements go here
+    get tableOne () { return $$('//*[@id="mw-content-text"]/div/table[1]/tbody') }
+    get precededBy () { return $(`${this.tableOne.selector}/tr[5]/td`) }
+    get surpassedBy () { return $(`${this.tableOne.selector}/tr[6]/td/a`) }
+    get allPhotos () { return $$('.magnify') }
```
Comment (m-schrepel) [inline test/pageobjects/empire.building.page.js:8]: This is really only grabbing the magnify icons themselves, not the photo containers on the page and checking if the .magnify class is a child of it.

File: test/specs/home.spec.js [hunk 2]
```diff
@@ -1,16 +1,57 @@
 import HomePage from '../pageobjects/home.page'
+import Custom from '../pageobjects/custom'
```
[the remark this answers is not in the document]
Comment (sumitbasra) [reply test/specs/home.spec.js]: 😄

File: test/pageobjects/custom.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import Page from './page'
+
+class Custom extends Page {
+    // Custom methods goes here
+    verifyMeterHeight (meter = null, ft = null) {
```
[the remark this answers is not in the document]
Comment (sumitbasra) [reply test/pageobjects/custom.js]: Updated. Thanks!

File: test/pageobjects/empire.building.page.js (hunk 1 above)
Comment (sumitbasra) [reply test/pageobjects/empire.building.page.js]: Updated correctly to grab photo container and then check for `.magnify`. First image does not have an icon so leaving that, there are 19 in length.

**Comment by m-schrepel** [comment]
There's one issue which is that the test should not pass for the image magnify. It's not a big deal but this image does not have the icon and your test isn't indicating that.

Screen Shot 2020-05-15 at 9 12 21 AM

**Comment by sumitbasra** [comment]
Merging this for now.

**Pull request merged**